src/components/Quote.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import '../styles/quote.css';
+
+class Quote extends Component {
+  render () {
+      return (
+        <section className="quotes">
+          <div className="quotes-container">
+            <div className="customer-blockquote">
```

Comment:
The repetition of the `customer-blockquote` implies we're missing a component.  This also makes me think that `Quote` should really be `Quotes`, whose data would be an array of quote data...